services/mailer.test.js

```diff
@@ -0,0 +1,19 @@
+const email = require('./mailer');
+const mailgun = require('mailgun-js');
+const mockSend = jest.fn(()=>Promise.resolve());
+
+jest.mock('mailgun-js', () => {
+    return jest.fn(() =>({
+        messages: ()=>({
+            send: mockSend
+        })
+    }))
+})
+
+describe('Test mailgun', ()=>{
+    it('mailgun', ()=>{
+        email()
+        expect(mockSend).toHaveBeenCalledTimes(1)
+        expect(mailgun).toHaveBeenCalledTimes(1)
+    })
```

Comment:
Add a test for argument. `mockSend` should have been called with arguments too!

Reply:
are you asking me to write a line of code at line 35 like this:

expect(mockSend)toHaveBeenCalledTimes(1)

Reply:
`expect(sendFn.mock.calls[0][1]).toEqual('paul@github.com')`
`expect(sendFn.mock.calls[0][2]).toEqual('token123')`